projects/project-player-library/src/lib/shared/task-card/task-card.component.html

```diff
@@ -1,47 +1,59 @@
 <mat-card class="items-card">
-    <div>
-        <div class="items-header" >
-            <span (click)="moveToDetailsTask(task._id)">
-                        <span class="complete-task" *ngIf="task.status !== 'completed'"></span>
-                        <mat-icon *ngIf="task.status === 'completed'"  style="color: green;" fontIcon="check_circle"></mat-icon>
-                <h4 >
-                    {{task.name}}
-                </h4>
-            </span>
-            <div>
-                <button mat-icon-button [matMenuTriggerFor]="menu" aria-label="Example icon-button with a menu">
-                    <mat-icon color="primary">more_vert</mat-icon>
-                  </button>
-                  <mat-menu #menu="matMenu" class="menu">
-                    @for (item of actionsList; track $index) {
-                        <button class="btn-menu"
-                                (click)="actionsEmit(item)"
-                                mat-menu-item
-                                style="padding: 0;"
-                                *ngIf="!submittedImprovement || (item.name !== 'DELETE' && item.name !== 'EDIT')">
-                            <div class="btn-menu-div">
-                                <span>{{item.name | translate}}</span>
-                                <span class="material-symbols-outlined">
-                                    <mat-icon aria-hidden="false" aria-label="Example edit icon" [fontIcon]="item.icon"></mat-icon>
-                                </span>
-                            </div>
-                        </button>
-                    }
-                  </mat-menu>
-            </div>
+    <div >
+        <div class="flex  w-full">
+            <div class="w-6 icon-complete mt-2">
+                <span class="complete-task" *ngIf="task.status !== 'completed'"></span>
```

Comment:
Instead of hardcoding the "completed" use if from the status.constants file

Reply:
done